### ISSUE TITLE
roxygenize deletes Rdfiles

### ISSUE THREAD

**Issue by brry** [r-lib/roxygen2#1287]
On my new setup, `roxygenize` works the first time, but then later deletes the Rd files.  
I tracked it down to roclet_output.roclet_rd <https://github.com/r-lib/roxygen2/blob/7b706c9f7f2e4227c98928a67c5dcb51d88ec15d/R/rd.R#L69>
which should be empty but isn't.  
`dir(man, full.names = TRUE`) returns "C:/path/pack/man/fun1.Rd",  "C:/R/pack/man/fun2.Rd" etc  
while `paths`, coming from `file.path(man, names(results))` is "C:\\\\path\\\\pack\\\\man/fun1.Rd" etc with double backslashes as path separators. (Bitten again. Why on earth did Windows choose this?!)

`man` comes from  `man <- normalizePath(file.path(base_path, "man"))` and the `base_path` input argument already had backslashes instead of normal slashes.

Basically, dir() should use double backslashes as path separator but uses forward and then roxygen fails.

**Comment by brry** [comment]
Rstudio fixed the issue on their end. I think we can close this issue.

**Comment by gaborcsardi** [comment]
@brry go ahead and close it then. :)
